controls/cis_ocp_1_4_0/section-1.yml

```diff
@@ -369,8 +369,9 @@ controls:
     controls:
     - id: 1.4.1
       title: Ensure that the healthz endpoints for the scheduler are protected by RBAC
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - scheduler_profiling_protected_by_rbac
```

Comment:
Technically - the audit procedures in the benchmark are slightly different to check that the service isn't exposed externally, but the check we have here doesn't really care about that difference. It's just checking the presence of an RBAC role.

Since the guidance is a little different, we could generate a new rule so it's accurate with the CIS control.

Reply:
> Technically - the audit procedures in the benchmark are slightly different to check that the service isn't exposed externally, but the check we have here doesn't really care about that difference. It's just checking the presence of an RBAC role.
 
Understood.
IIUC, the check done by the Compliance Operator is the same for both rules, but the manual check is different.

> Since the guidance is a little different, we could generate a new rule so it's accurate with the CIS control (unless we can override it?).

By "overriding" you mean to generate another rule based off of this one? The project doesn't have this capability.
So we would need a new rule.